packages/contractkit/src/utils/tx-uri.ts

```diff
@@ -1,11 +1,12 @@
 import { trimLeading0x } from '@celo/utils/lib/address'
 import BN from 'bn.js'
-import { range } from 'lodash'
 import qrcode from 'qrcode'
 import querystring from 'querystring'
 import { Tx } from 'web3-core'
 import abi from 'web3-eth-abi'
 
+const range = (to: number) => Array.from(Array(to).keys())
```

Comment:
can we have a `range()` exported in some utitily pkg, inside contractkit at least?

Reply:
Done!